index.js

```diff
@@ -1 +1 @@
-require("style!css!sass!./font-awesome-styles!./font-awesome.config.js");
+require("style!css!sass!./font-awesome-styles.loader!./font-awesome.config.js");
```

Comment:
This still doesn't work. I had to update to:

`require("style-loader!css-loader!sass-loader!./font-awesome-styles.loader!./font-awesome.config.js");`

Then it compiles but it still doesn't actually work. The fonts are never downloaded.